Assets/Mirror/Core/PortTransport.cs

```diff
@@ -0,0 +1,17 @@
+// convenience interface for transports which use a port.
+// useful for cases where someone wants to 'just set the port' independent of
+// which transport it is.
+//
+// note that not all transports have ports, but most do.
+
+namespace  Mirror
+{
+    public interface PortTransport
```

Comment:
I'd prefer IPortTransport for an interface, not sure what we do for naming though

Reply:
I agree, IPortTransport would be a lot better

Reply:
me too interfaces need to start with I

Reply:
I agree, Interface is best named with 'I' -> IPortTransport